.github/workflows/deploy.yml

```diff
@@ -12,6 +12,10 @@ jobs:
     strategy:
       matrix:
         container: [ "ubuntu:18.04", "ubuntu:19.10", "ubuntu:20.04", "debian:9", "debian:10", "fedora:29", "fedora:30", "fedora:31", "fedora:32", "archlinux:latest" ]
```

Comment:
I really want to remove support for 18.04 & 19.10. But I fear that we'll have to maintain support for 18.04 for some time and If canonical moves it into the archived list in the meantime?

Reply:
ubuntu 18 is an LTS release it's officially supported until 2023!

Reply:
Then there is no need to drop these two!